common/internal-types/src/channels.rs

```diff
@@ -326,4 +340,14 @@ mod tests {
             ce.remaining_closure_time(current_time).expect("must have closure time")
         );
     }
+
```

Comment:
Missing tests `ChannelStatus::from_str(&ChannelStatus::Open.to_string())` ...etc.
To ensure consistency with the `Display` trait impl.

Reply:
added